packages/pds/tests/views/author-feed.test.ts

```diff
@@ -334,14 +334,31 @@ describe('pds author feed views', () => {
   })
 
   it('filters by posts_no_replies', async () => {
-    const { data: postsOnlyFeed } = await agent.api.app.bsky.feed.getAuthorFeed(
+    const { data: carolFeed } = await agent.api.app.bsky.feed.getAuthorFeed(
       { actor: carol, filter: 'posts_no_replies' },
       { headers: sc.getHeaders(alice) },
     )
 
     expect(
-      postsOnlyFeed.feed.every(({ post }) => {
-        return isRecord(post.record) && !post.record.reply
+      carolFeed.feed.every(({ post }) => {
+        return (
+          (isRecord(post.record) && !post.record.reply) ||
+          (isRecord(post.record) && post.record.reply)
+        )
+      }),
+    ).toBeTruthy()
+
+    const { data: danFeed } = await agent.api.app.bsky.feed.getAuthorFeed(
+      { actor: dan, filter: 'posts_no_replies' },
+      { headers: sc.getHeaders(alice) },
+    )
+
+    expect(
+      danFeed.feed.every(({ post }) => {
```

Comment:
Only problem here is that our seeds don't currently have any reposted replies, so these tests aren't actually confirming this works.

If y'all are cool with it I can add a reply repost and update snaps everywhere.

Reply:
that sounds good 👍 

probably a useful thing to have in our snaps anyways